<launch>
  <!--arg name="image_topic_name" value="/ptgrey_simple/pgr_mono" /-->
  <!-- remap output topics -->
  <arg name="image_topic_name" value="/droni/image_raw" />
  <arg name="width" default="640" />
  <arg name="height" default="480" />

  <node pkg="nodelet" type="nodelet" name="standalone_nodelet" args="manager" output="screen"/>

  <param name="/image_topic_name" type="string" value="$(arg image_topic_name)" />
  <param name="resize_width" value="$(arg width)" />
  <param name="resize_height" value="$(arg height)" />

  <!--node pkg="nodelet" type="nodelet" name="ptgrey_simple" args="load ptgrey_simple/ptgrey_simple standalone_nodelet" output="screen">
</node-->
  <include file="$(find flea3)/launch/single_nodelet.launch"/>

  <param name="/image_topic_name" type="string" value="$(arg image_topic_name)" />
  <node pkg="nodelet" type="nodelet" name="fakecam" args="load fakecam/fakecam standalone_nodelet" output="screen">
      <rosparam command="load" file="$(find fakecam)/launch/fakecam.yaml" />
      <param name="framerate" value="30.0" />
      <param name="publish_slower_image" value="3" />
  </node>

  <!-- remap input topics -->
  <!--remap from = "image" to = "$(arg image_topic_name)_slow" />
  <node pkg="nodelet" type="nodelet" name="imageview" args="load image_view/image    standalone_nodelet"  output="screen">
      <param name="autosize" value="false" />
  </node-->

</launch>
